Userland/Libraries/LibWeb/Page/Page.h

```diff
@@ -34,6 +34,7 @@
 #include <LibWeb/HTML/SelectItem.h>
 #include <LibWeb/Loader/FileRequest.h>
 #include <LibWeb/PixelUnits.h>
+#include <LibWebView/SocketPair.h>
```

Comment:
hmm I guess this technically works since we only need the header-declared struct here, but in general LibWeb cannot depend on LibWebView. Just something to keep in mind if this becomes a file we need to actually link in the future.

Reply:
It already does depend on it in the same way for the WebView::Attributes, no? Or should I create a new type that handles the same thing in LibWeb 🤔

Reply:
`WebView::Attributes` is between LibWebView and WebContent only, doesn't leak into LibWeb. I think `WebView::SocketPair` could live in LibWeb? But not the end of the world right now since it's a header-only thing currently.